knownprojects_build/_sql/_procedures.sql

```diff
@@ -77,10 +80,13 @@ BEGIN
     WHERE table_schema = 'public' AND table_name = _table INTO _valid_fields;
 
     FOR _record_id, _field, _old_value, _new_value IN 
-        SELECT record_id, field, old_value, new_value 
-        FROM corrections_main WHERE field=any(_valid_fields)
+        EXECUTE FORMAT($n$
+            SELECT record_id, field, old_value, new_value 
+            FROM %1$s
+            WHERE field = any(%2$L)
```

Comment:
Why does _valid_fields need to get formatted here but didn't need to before?

Reply:
there's no EXECUTE FORMAT previously on this statement, we hard coded the corrections table as corrections main